apps/ledger-live-mobile/.env.mock

```diff
@@ -8,6 +8,7 @@ ADJUST_APP_TOKEN=cbxft2ch7wn4
 BRAZE_ANDROID_API_KEY="be5e1bc8-43f1-4864-b097-076a3c693a43"
 BRAZE_IOS_API_KEY="e0a7dfaf-fc30-48f6-b998-01dbebbb73a4"
 BRAZE_CUSTOM_ENDPOINT="sdk.fra-02.braze.eu"
-FEATURE_FLAGS={"syncOnboarding":{"enabled":false},"llmNewDeviceSelection":{"enabled":false},"protectServicesMobile":{"enabled":false},"brazePushNotifications":{"enabled":false}}
+FEATURE_FLAGS={"syncOnboarding":{"enabled":false},"llmNewDeviceSelection":{"enabled":false},"protectServicesMobile":{"enabled":false},"brazePushNotifications":{"enabled":false},"discover":{"enabled":true,"params":{"version":"2"}}}
```

Comment:
I think we should try removing this. I don't think it's necessary for navigating to local live apps via deeplinks so best to avoid any side effects for other functionality.